site/content/en/docs/Integration Patterns/canary-testing.md

```diff
@@ -35,12 +35,17 @@ state that in our system, the label `canary: "true"` will be added to any canary
 apiVersion: "allocation.agones.dev/v1"
 kind: GameServerAllocation
 spec:
-  preferred:
-    - matchLabels:
-        canary: "true"
-  required:
-    matchLabels:
+  - matchLabels:
+      canary: "true"
       agones.dev/fleet: stable
+    players:
```

Comment:
We can remove the "players" selector. (Apologies I realize it's probably a bit confusing from the original issue.)

Reply:
I have removed the `players` selector.